pallets/liquid-staking/src/lib.rs

```diff
@@ -1040,6 +1051,14 @@ pub mod pallet {
             let account_id = Self::account_id();
             let asset: MultiAsset = (MultiLocation::here(), fees).into();
 
+            log::trace!(
+                target: "liquidstaking::ump_transact",
+                "call: {:?}, asset: {:?}, xcm_weight: {:?}",
```

Comment:
have you tested if `call` is human readable because if it's not then it wont be very useful

Reply:
It will be helpful too if it's a raw bytes vec, remember last time I compare two bytes vec between parachain xcm message and relaychain xcm.

Reply:
ok we can do this first I think